# Independent sample z-test

[Independent Two-sample z-test](http://facweb.cs.depaul.edu/sjost/csc423/documents/test-descriptions/indep-z.pdf "Independent Two-sample z-test")


Assumptions

• Experimental Design: The sample forms two independent treatment
groups.

• Null Hypothesis: The population means of the two treatment groups
are not significantly different from each other.

• Population Distribution: Arbitrary distribution within each treatment group.

• Sample Size: Size of each treatment group is equal to or greater than
30.


## Case 1 two tailed z test

Among the six-year-old children in a certain area, 30 boys were randomly selected, with an average height of 114cm. 27 girls were selected, with an average height of 112.5cm. According to previous data, the standard deviation of height of six-year-old boys and girls in this area is 5cm for boys and 5cm for girls 6.5cm, ask if there is any significant difference in height between boys and girls aged 6 in this area? (α = 0.05)


H0 : ${\mu_0}$ ＝ ${\mu_1}$
<br/>
H1 : ${\mu_0}$ $\not=$ ${\mu_1}$

$\bar{X_1}$ = 114, $\bar{X_2}$ = 112, $\sigma_1$ = 5, $\sigma_2$ = 6.5, $n_1$ = 30, $n_1$ = 27

z = $\frac{\bar{X_1}-\bar{X_2}}{ \sqrt{\frac{ \sigma_1^2}{n_1}+ \frac{ \sigma_2^2}{n_2}} }$ = $\frac{114-112.5}{ \sqrt{\frac{ 5^2}{30}+ \frac{ 6.5^2}{27}} }$

In [4]:
import math
z = (114 - 112.5) / math.sqrt((math.pow(5, 2)/30)+(math.pow(6.5, 2)/27))
print(z)

0.9686196045011365



α/2 = 0.025, look up table $Z_{\alpha/2}$ = 1.96

z = 0.97 < 1.96, so accept H0, so there is no significant difference in height between boys and girls